src/App.IntegrationTests/ApiTests/StatelessDataApiTest.cs

```diff
@@ -56,6 +55,44 @@ public async Task StatelessData_Get_ObjectSucessfullyPrefilledAndCalculated()
             Assert.Equal(expectedCalculatedValue, actualCalculatedValue);
         }
 
+        [Fact]
+        public async Task GetAnonymous_CorrectDataType_ShouldReturnOk()
+        {
+            // Arrange
+            string org = "ttd";
+            string app = "anonymous-stateless";
+            HttpClient client = SetupUtil.GetTestClient(_factory, org, app);
+            string requestUri = $"/{org}/{app}/v1/data/anonymous?dataType=Veileder";
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, requestUri);
+
+            // Act
+            HttpResponseMessage res = await client.SendAsync(httpRequestMessage);
+            string responseContent = await res.Content.ReadAsStringAsync();
+            Mocks.Apps.Ttd.AnonymousStateless.Models.Veileder dataObject = JsonConvert.DeserializeObject<Mocks.Apps.Ttd.AnonymousStateless.Models.Veileder>(responseContent);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, res.StatusCode);
+        }
+
+        [Theory]
+        [InlineData("nonexisting")]
+        [InlineData("")]
+        public async Task GetAnonymous_CorrectDataType_ShouldReturnBadRequest(string dataType)
```

Comment:
Copy/Paste error? Testname name suggest correct dataType should return bad request 
```suggestion
        public async Task GetAnonymous_BadDataType_ShouldReturnBadRequest(string dataType)
```

Reply:
Thx, copy/paste error:)